mbuild/formats/lammpsdata.py

```diff
@@ -271,15 +321,19 @@ def write_lammpsdata(structure, filename, atom_style='full',
 
 
         data.write('\n')
+        #If using real units, write our box dimensions in angstrom instead of nm.
+        boxdim_conversion = 10.0
+        if unit_style == 'lj':
```

Comment:
I may have done this incorrectly originally.  Should the box dimensions be the distance / the sigma conversion factor?

Reply:
I think you did it several lines above, at line 222. That's why you specified the conversion to be 1.0 here for lj.